### PR TITLE
Fix loader removing tabs from DOM issue

### PR DESCRIPTION
**Pull request by ashwanijha1996**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: frontend/src/components/Loading/_Loading.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .loading {
-    
+
```
Comment (furkanmutlu) [inline frontend/src/components/Loading/_Loading.scss:2]:
```suggestion
```

**Comment by ashwanijha1996** [comment]
Hi @furkanmutlu  i have closed last PR and raised this new PR with 1 commit only. Please review and tell me if it is fine or not?

**Comment by furkanmutlu** [comment]
> Hi @furkanmutlu i have closed last PR and raised this new PR with 1 commit only. Please review and tell me if it is fine or not?

sure, that should work fine as well, but I would strongly recommend you to learn `git reset`. That would be helpful for you.

**Review by furkanmutlu — APPROVED** [review]
lgtm!

**Comment by ashwanijha1996** [comment]
Ok sure i have started my journey and i will surely learn git concepts. Thank you very much for you help and support.

**Comment by furkanmutlu** [comment]
> Ok sure i have started my journey and i will surely learn git concepts. Thank you very much for you help and support.

Thank you very much for your contribution as well, looking for more for the future! keep in mind that I would always help as much as I could. ;)

**Pull request merged**